build_debian.sh

```diff
@@ -527,9 +530,6 @@ then
 
 fi
 
-## Add mtd and uboot firmware tools package
-sudo LANG=C chroot $FILESYSTEM_ROOT apt-get -y install u-boot-tools mtd-utils device-tree-compiler
-
```

Comment:
why move?

Reply:
This is for the compile fail:
```
Processing triggers for initramfs-tools (0.133) ...
update-initramfs: Generating /boot/initrd.img-4.19.0-12-2-arm64
E: /etc/initramfs-tools/hooks/uboot-utils failed with return 1.
update-initramfs: failed for /boot/initrd.img-4.19.0-12-2-arm64 with 1.
dpkg: error processing package initramfs-tools (--configure):
 installed initramfs-tools package post-installation script subprocess returned error exit status 1
Errors were encountered while processing:
 initramfs-tools
E: Sub-process /usr/bin/dpkg returned an error code (1)
+ sudo LANG=C chroot ./fsroot umount /proc
```

The script files/initramfs-tools/uboot-utils will be called during update-initramfs.  uboot-utils is depend on /usr/sbin/ubiattach and /usr/sbin/mtdinfo which are installed by mtd-utils.